code/data_quality_tab/data_monitoring_completeness.R

```diff
@@ -11,6 +11,26 @@ library(purrr)
 library(lubridate)
 
 
+
+# Load Function(s) --------------------------------------------------------
+
+#outputs data frame with completeness percentage per data item
+source(here::here("functions", "completeness.R"))
+
+#binds the df outputs from completeness() and appends the df names in a 
+#new column called source
+source(here::here("functions", "append_source.R"))
+
+#takes the output from append_source() and outputs symbols indicating trend
+#in completeness percentages compared to previous month
+source(here::here("functions", "create_change_symbol.R"))
+
+#takes the output from append_source() and outputs symbols indicating range of
+#completeness percentage values, data items where percentage too low are flagged.
+source(here::here("functions", "create_flag_symbol.R"))
```

Comment:
All of this can be exchanged with a line of code. This uses the `walk()` function from the `purrr` package, so that will need to be loaded along with the other packages for this script (or in a `setup_environment` script if that is going to be used for all scripts in the code folder.

```suggestion
library(purrr)

# Load Function(s) --------------------------------------------------------
walk(list.files(here("functions"), full.names = TRUE), source)
```